src/internal/connector/onedrive/collections.go

```diff
@@ -228,7 +227,17 @@ func deserializeMetadata(
 		}
 	}
 
-	return prevDeltas, prevFolders, el.Failure()
+	// if reads from items failed, return empty but no error
+	if errs.Failure() != nil {
+		// TODO(meain): Should we consider failing only if we fail to
+		// read `previouspath` and use the metadata if we just fail on
+		// `delta`?
```

Comment:
As of now, if we fail to fetch any metadata item, we revert to a non-incremental backup. We could maybe ignore failing to pull `delta`?

Reply:
let's go with this for now. It's alright to be a bit conservative and we can come back and loosen the restrictions later on